content/docs/concepts/traffic-management/traffic-routing/index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Traffic Routing
```

Comment:
I'm not sure about this title. Most users will think of what you do with Virtual Services as traffic routing (ie specifying exactly which destination services you want to get which traffic), which is not what this doc is about. Service subsets? Grouping services?